SubmittingPatches.rst

```diff
@@ -133,7 +133,7 @@ Some negative examples (how *not* to title a commit message)::
      bug fix for driver X
      fix issue 99999
 
-Further to the last negative example ("fix issue 99999"), see `Fixes line`_.
+Further to the last negative example ("fix issue 99999"), see :ref:`fixes line`.
```

Comment:
I have tried both 'fixes line'_ and :ref:'fixes line'. Both don't work. We
cannot use ref as anchor is defined in later part. I am reverting back to
original commit.